### PR TITLE
docs: grammatical changes to how-to-work-on-coding-challenges

### PR DESCRIPTION
**Pull request by moi238**
Made some minor grammatical changes to improve the flow of the sentences.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: docs/how-to-work-on-coding-challenges.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ If it takes longer than two minutes to complete the challenge, you have two opti
 
 The 2-minute rule forces you, the challenge designer, to make your directions concise, your seed code clear, and your tests straight-forward.
 
-We have JavaScript events that track how long it takes for campers to solve challenges and we can use them to identify challenges that need to be simplified or split.
+We have JavaScript events that track how long it takes for campers to solve challenges, and we can use them to identify challenges that need to be simplified or split.
```
Comment (RandellDawson) [inline docs/how-to-work-on-coding-challenges.md]: Suggested change:
```suggestion
We track how long it takes for campers to solve changes and use this information to identify challenges that need to be simplified or split.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@moi238 Please see several suggested changes to your PR.  You were adding several unnecessary commas.

**Review by moi238 — COMMENTED** [review]
All changes made as requested.

**Comment by RandellDawson** [comment]
Waiting to merge until PR #37103 and #37238 have been reviewed/merged/closed first.

**Comment by moT01** [comment]
Fixed conflicts on behalf of user.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**